mid-assessment/q7.rb

```diff
@@ -0,0 +1,14 @@
+# Implement a program to check if a given string is a palindrome or not.
+def pallindrome?
+  print 'Enter a string: '
+  pall = gets.chomp
+  len = pall.length
+  pall_rev = ''
+  i = len - 1
+  while i >= 0
+    pall_rev << pall[i]
+    i -= 1
+  end
+  pall == pall_rev
+end
+puts pallindrome?
```

Comment:
Also 1 corner case is missed. If i do not enter any string then it returns true. But it should print "Please enter valid string"

Reply:
updated corner case